src/dataloader.py

```diff
@@ -0,0 +1,100 @@
+import os
+import zipfile
+import tarfile
+import shutil
+from pathlib import Path
+from torch.utils import data
+from PIL import Image
+
+def UnZipFolder(file_path, folder_inside, destination_path):
```

Comment:
I see this function as unnecessary. Maybe it could be interesting to put it inside the "utils.py" file. Even so, keep in mind that in the data folder we will have the files already decompressed. Do not include this function in dataloader.py

Reply:
Only as function in Utils not used